### PR TITLE
Add 'sellonbid' for Livecoin

### PR DESCRIPTION
**Pull request by Tristian**
#68

### CODE REVIEW

File: web/yaamp/core/trading/livecoin_trading.php [hunk 1]
```diff
@@ -177,13 +181,13 @@ function doLiveCoinTrading($quick = false)
...
 
 	foreach ($balances as $balance) {
-		if ($balance->type != 'total') {
+		if ($balance->type != 'available') {
 			continue;
 		}
 
-		$amount = $balance->value;
+		$balance = $balance->value;
 		$symbol = $balance->currency;
```
Comment (tpruvot) [inline web/yaamp/core/trading/livecoin_trading.php]: there is a problem there

**Comment by Tristian** [comment]
Fixed it before I saw your comment. I should probably stop skipping meals...
Thanks.

File: web/yaamp/core/trading/livecoin_trading.php [hunk 2]
```diff
@@ -228,7 +228,7 @@ function doLiveCoinTrading($quick = false)
 			sleep(1);
 
 			$res = $livecoin->sellLimit($pair, $sellprice, $balance);
-			if (!$res->success == 'true' && $res->added == 'true')) {
+			if (!$res->success == 'true' && $res->added == 'true') {
```
Comment (tpruvot) [inline web/yaamp/core/trading/livecoin_trading.php]: are you sure its a string ?

https://www.livecoin.net/api/orders#exchangeselllimit

**Comment by Tristian** [comment]
I'm really not sure of anything anymore.
It does look like it should be a boolean, not a string.

**Pull request merged**